### PR TITLE
Nerdstore bugfix

### PR DESCRIPTION
**Pull request by tangollama**
Removing the autosizer and addressing a few issues with status pages.

### CODE REVIEW

**Comment by ericSpence** [comment]
Looks good to me. Feel free to merge this.

**Comment by ericSpence** [comment]
looks like when I try to run npm start I hit a cli issue
```
  Launchers:
    ✖ status-page-launcher launchers/status-page/nr1.json
      └ The property icon is required.
      └ Property /rootNerdletId The id must be lowercase, and can only contain alphabetic characters, numbers, hyphens and a dot between the package id and the artifact id. For example: 'example-package.my-nerdlet-id'. It should match the regular expression: ^([a-z0-9]+[a-z0-9-]*)\.([a-z0-9]+[a-z0-9-]*)$.
```

File: launchers/status-page/nr1.json [hunk 1]
```diff
@@ -3,5 +3,5 @@
     "id": "status-page-launcher",
     "description": "Review all the relevant status pages in your world as track through the incident reports.",
     "displayName": "Status Pages",
-    "rootNerdletId": "status-page"
```
Comment (ericSpence) [inline launchers/status-page/nr1.json]: Is there a current change in flight for nr1? currently the cli is telling me that this has to be ```<string>.<string>``` and refuses to start

Comment (ericSpence) [reply launchers/status-page/nr1.json]: I have a fix done locally but if a change is incoming I won't push it up

**Comment by tangollama** [comment]
You’re probably still running the EAP CLI. You need to blow away and install the new CLI. 

After that, future updates should be handled by an nr1 update command.

**Comment by ericSpence** [comment]
Yeah that looks to have resolved my issue. 👍 good call on that

**Comment by ericSpence** [comment]
The rest look good to me feel fee to merge it

**Pull request merged**